recipe/meta.yaml

```diff
@@ -42,19 +33,42 @@ outputs:
       run:
         - {{ pin_subpackage('beautifulsoup4', max_pin="x.x.x") }}
     test:
+      source_files:
+        - bs4/tests
       imports:
         - bs4
+      requires:
+        - pip
+        - pytest
+      commands:
+        - pip check
+        - pytest bs4 -vv  
+
+test:
+  source_files:
+    - bs4/tests
```

Comment:
See above comment about source files in the test section.

Reply:
I've removed `source_files`